Dockerfile.openshift

```
@@ -1,24 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS rhel9
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.26-openshift-5.0 AS builder
 ADD . /go/src/github.com/openshift/egress-router-cni
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)